src/appState.ts

```diff
@@ -53,7 +53,10 @@ export const getDefaultAppState = (): Omit<
     isResizing: false,
     isRotating: false,
     selectionElement: null,
-    zoom: 1,
+    zoom: {
+      value: 1,
+      translation: { x: 0, y: 0 },
+    },
```

Comment:
we'll need a migration for this in `restore.ts`

Reply:
I'll take a look at it 👍

Reply:
Should be taken care of now. Tested locally, and it worked fine.